libmscore/element.h

```diff
@@ -40,6 +40,7 @@ class MuseScoreView;
 class Segment;
 class TextStyle;
 class Element;
+class Annotation;
```

Comment:
These are things left over from the previous implementations. Let me close
this PR and make a new one after cleaning up the code.

On Tue, Aug 9, 2016 at 9:09 PM, Nicolas Froment <notifications@github.com>
wrote:

> In libmscore/element.h
> <https://github.com/musescore/MuseScore/pull/2762#discussion_r74083774>:
>
> > @@ -40,6 +40,7 @@ class MuseScoreView;
> >  class Segment;
> >  class TextStyle;
> >  class Element;
> > +class Annotation;
>
> ????
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/musescore/MuseScore/pull/2762/files/ec138f4e7db81ddd13520c9f8121ac0b497f2ff8#r74083774>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGf7ytLizUcHl_Na_UUZyc9hMsEm7esCks5qeJ8jgaJpZM4JZRIR>
> .
>



-- 
Ruchit Agrawal.
IIIT Hyderabad.